reference/commands/misc/user.rst

```diff
@@ -8,7 +8,9 @@ conan user
 
     $ conan user [-h] [-c] [-p [PASSWORD]] [-r REMOTE] [-j JSON] [-s] [name]
 
-Authenticates against a remote with user/pass, caching the auth token.
+Authenticates against a remote with user/pass, after a successful login the auth
```

Comment:
This text is usually the same as the one displayed in the `conan user --help`. I'd suggest to add it to the command line help or relate to it later in this section

Reply:
Didn't realize about it 🤦